code/modules/projectiles/projectile/bullets.dm

```diff
@@ -294,6 +294,47 @@
 	embed = 0
 	sharp = 0
 
+/obj/item/projectile/bullet/pistol/papadov

```

Comment:
This being its own object is pointless, it can just use the regular bullet type and there will be no difference.